webpack.config.js

```diff
@@ -173,8 +175,8 @@ var webpackConfig = {
             loader: "file-loader",
             options: {
               name: "[name].[ext]",
-              publicPath: "../../assets/svg",
-              outputPath: "../../assets/svg"
+              publicPath: "../assets/svg",
```

Comment:
hi @pookmish Do you see where I went wrong here? It puts it in the right /dist/ dir but is wrong in the css? I'm puzzled.

Reply:
one setting is for where to put the file and the other is for what path to write out in the css as the path to the file may be different from where the css is outputted and the webpack config file.

Reply:
Thank you. Those ../ are a hazard for me. 🍭